parlai/core/torch_agent.py

```diff
@@ -47,7 +47,7 @@
                       each row in the batch.
 """
 Batch = namedtuple('Batch', ['text_vec', 'text_lengths', 'label_vec',
-                             'label_lens', 'labels', 'valid_indices',
+                             'label_lengths', 'labels', 'valid_indices',
```

Comment:
please change in the definition as well (one of the `:field`s) above

Reply:
Good call! Was changing one to match the other but forgot to check which was in the comment.